src/pages/docs/custom-layouts.mdx

```diff
@@ -0,0 +1,176 @@
+---
+name: Custom Layouts 
+route: /docs/custom-layouts
+---
+
+import { Default, WithCustomRenderer } from './components/custom-layouts'
+
+# Custom layouts
+
+ The default layouts of JSON Forms are a good fit for most scenarios,
+but there might be certain situations where you'd want to customize the rendered layouts.
+JSON Forms allows for this by registering a custom renderer that produces a different UI
+for a given layout.
+
+In this section you will learn how to create and register a custom renderer for a layout.
+We will replace the default renderer for groups. By default a group is rendered like this:
+
+<Default />
+
+Our goal is to instead render the UI for groups as depicted below:
+
+<WithCustomRenderer />
+
+## Running the seed
+
+If you want to follow along with this tutorial, you may want to [clone the seed repository](https://github.com/eclipsesource/jsonforms-react-seed)
+which basically is just a skeleton application scaffolded by create-react-app with redux
+and JSON Forms dependencies added.
+
+```bash
+cd jsonforms-react-seed
+npm install
+npm start
+```
+
+Once the dependencies are installed and the local server has been started,
+navigate to http://localhost:3000 in order to see the application running.
+
+The seed is described in more detail within the `README.md` file of
+the [repo](https://github.com/eclipsesource/jsonforms-react-seed), hence we
+only focus on the most crucial parts of the app in the following.
+
+##  Core concepts about rendering
+
+Before explaining how to contribute a component (which will be referred to as "custom layout") to JSON Forms,
+we first explain how the basic process of rendering works.
+
+JSON Forms maintains a registry of renderers (which are regular React components). When JSON Forms is instructed to
```

Comment:
(in this tutorial we use the React/Material renderers producing regular React components)